sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/SpannerChangeStreamErrorTest.java

```diff
@@ -311,11 +351,9 @@ public void testInvalidRecordReceived() {
       pipeline.run().waitUntilFinish();
     } finally {
       thrown.expect(PipelineExecutionException.class);
-      // DatabaseClient.getDialect returns "DEADLINE_EXCEEDED: Operation did not complete in the "
-      // given time" even though we mocked it out.
       thrown.expectMessage("Field not found");
       assertThat(
-          mockSpannerService.countRequestsOfType(ExecuteSqlRequest.class), Matchers.equalTo(0));
+          mockSpannerService.countRequestsOfType(ExecuteSqlRequest.class), Matchers.greaterThan(0));
```

Comment:
This is a behavior change from expecting 0 to expecting great than 0. Just noting this. I don't have a strong opinion about it.

Reply:
I think the behavior change is fine, since previously this test was disabled, so I'm not exactly sure which part was broken.